src/wam2layers/utils/grid.py

```diff
@@ -0,0 +1,68 @@
+"""Functionality related to the grid."""
+
+import numpy as np
+import xarray as xr
+
+
+def stagger_x(f):
+    """Interpolate f to the grid cell interfaces.
+    Only the values at the interior interfaces are returned
+    Arguments:
+        f: 2d array of shape [M, N]
+    Returns:
+        2d array of shape [M-2, N-1]
+    """
+    return 0.5 * (f[:, :-1] + f[:, 1:])[1:-1, :]
+
+
+def stagger_y(f):
+    """Interpolate f to the grid cell interfaces.
+    Only the values at the interior interfaces are returned
+    Arguments:
+        f: 2d array of shape [M, N]
+    Returns:
+        2d array of shape [M-1, N-2]
+    """
+    return 0.5 * (f[:-1, :] + f[1:, :])[:, 1:-1]
```

Comment:
Aren't you neglecting the change in gridcell width (km) as you move north/south here? This is not a conservative regridding step. You probably have a net loss in moisture due to this step, assuming closer to the equator there is more moisture.

(Is this accounted for at all in wam2layers?)

Reply:
If this was a problem before (interpolating between m3/timestep fluxes), then it's not a problem anymore as we are now interpolating kg m-1 s-1, which is not dependent on the shape of the grid cells